web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -1321,6 +1321,18 @@ class LiftRules() extends Factory with FormVendor with LazyLoggable {
   val snippets = RulesSeq[SnippetPF]
 
   /**
+   * Handles the parsing of template content into NodeSeqs.
+   */
+  @volatile var contentParsers: Seq[ContentParser] = Seq(
+    ContentParser.full(
+      Seq("html", "xhtml", "htm"),
+      S.htmlProperties.htmlParser,
+      content => content // These templates are not surrounded  by default
```

Comment:
Maybe make this a named parameter and use `identity` instead of `content=>content`?